### PR TITLE
[-] FO : Fix #PSCSX-5182, for newsletter flag for customer if previously registred as guest

### PR DESCRIPTION
**Pull request by gRoussac**
…sly registred as guet

### CODE REVIEW

**Comment by gRoussac** [comment]
NEEDS TO BE MERGED AFTER https://github.com/PrestaShop/blocknewsletter/pull/15

File: controllers/front/AuthController.php [hunk 1]
```diff
@@ -359,15 +359,22 @@ protected function processSubmitLogin()
      */
     protected function processCustomerNewsletter(&$customer)
     {
+        $blocknewsletter = Module::isInstalled('blocknewsletter') && $module_newsletter = Module::getInstanceByName('blocknewsletter');
+        if ($blocknewsletter && $module_newsletter->active && !Tools::getValue('newsletter')) {
+            if ($module_newsletter->isNewsletterRegistered2($_POST['email']) == $module_newsletter::GUEST_REGISTERED) {
```
Comment (tchauviere) [inline controllers/front/AuthController.php]: Tools::getValue() here ?

**Comment by gRoussac** [comment]
done :guardsman:

**Comment by tchauviere** [comment]
:+1:

**Pull request merged**